diff --git a/node_modules/fs-extra/lib/ensure/file.js b/node_modules/fs-extra/lib/ensure/file.js
index 67eed30..55c75e7 100644
--- a/node_modules/fs-extra/lib/ensure/file.js
+++ b/node_modules/fs-extra/lib/ensure/file.js
@@ -8,7 +8,8 @@ const pathExists = require('../path-exists').pathExists
 
 function createFile (file, callback) {
   function makeFile () {
-    fs.writeFile(file, '', err => {
+    console.log(`Going to create empty file/dir ${file}`)
+    fs.writeFile(file, ' ', err => {
       if (err) return callback(err)
       callback()
     })
